tests/unittests/sources/helpers/test_netlink.py

```diff
@@ -130,41 +132,38 @@ def test_read_invalid_rta_ifname_none(self):
             "HHc", buf, RTATTR_START_OFFSET, 5, 16, int_to_bytes(OPER_DOWN)
         )
         interface_state = read_rta_oper_state(buf)
-        self.assertIsNone(interface_state)
+        assert interface_state is None
 
     def test_read_invalid_data_len(self):
         """raise assert error if data size is smaller than required size"""
         buf = bytearray(32)
-        with self.assertRaises(AssertionError) as context:
+        with pytest.raises(AssertionError) as excinfo:
             read_rta_oper_state(buf)
-        self.assertTrue(
-            "length of data is smaller than RTATTR_START_OFFSET"
-            in str(context.exception)
+        assert "length of data is smaller than RTATTR_START_OFFSET" in str(
+            excinfo.value
         )
 
     def test_unpack_rta_attr_none_data(self):
         """unpack_rta_attr raises assert error if data is none"""
         data = None
-        with self.assertRaises(AssertionError) as context:
+        with pytest.raises(AssertionError) as excinfo:
             unpack_rta_attr(data, RTATTR_START_OFFSET)
-        self.assertTrue("data is none" in str(context.exception))
+        assert "data is none" in str(excinfo.value)
 
     def test_unpack_rta_attr_invalid_offset(self):
         """unpack_rta_attr raises assert error if offset is invalid"""
         data = bytearray(48)
-        with self.assertRaises(AssertionError) as context:
+        with pytest.raises(AssertionError) as excinfo:
             unpack_rta_attr(data, "offset")
-        self.assertTrue("offset is not integer" in str(context.exception))
-        with self.assertRaises(AssertionError) as context:
+        assert "offset is not integer" in str(excinfo.value)
+        with pytest.raises(AssertionError) as excinfo:
```

Comment:
If you like too using `match=` for these two calls may allow collapsing each into single lines